### PR TITLE
Expand npc snippets

### PR DESCRIPTION
**Pull request by Stone94**
Added some new snippets over several categories, took some grammatical liberties for dialogue pieces.

#### Summary
None

#### Purpose of change
Increase the variety of NPC speech lines

#### Describe the solution
Added extra snippets in speech_tags.json

#### Describe alternatives you've considered
None

#### Testing
Loaded up the game, spawned some NPCs, and went to the refugee center, snippets appeared, and no errors occured.

### CODE REVIEW

**Comment by Stone94** [comment]
I assume the build test is failing because I didn't double space after the ? And ! between phrases?

**Review by I-am-Erk — COMMENTED** [review]
There's a few more but this is a start

File: data/json/npcs/talk_tags.json [hunk 1]
```diff
@@ -549,7 +575,11 @@
       "There's a bed calling my name, and I'm going to it.",
       "Good night!  Wake me if you need me.",
       "Well, I'm going to bed now.",
-      "Calling it a night for now.  You get some rest too, okay?"
+      "Calling it a night for now.  You get some rest too, okay?",
+      "I'm getting some shut-eye, good night!",
+      "Time to view the old eye-lid cinema, good night!",
+      "Time to hit the hay",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:581]:
```suggestion
      "Time to hit the hay.",
```

File: data/json/npcs/talk_tags.json [hunk 2]
```diff
@@ -549,7 +575,11 @@
...
+      "I'm going to see about a bed with my name on it, good night"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:582]:
```suggestion
      "I'm going to see about a bed with my name on it, good night."
```

File: data/json/npcs/talk_tags.json [hunk 3]
```diff
@@ -636,7 +667,11 @@
       "Out of my way, <name_b>!",
       "Move it, <name_g>!",
       "You need to move, <name_g>, <okay>?",
-      "Coming through!"
+      "Coming through!",
+      "Out of my <swear> way!",
+      "I need to get by",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:672]:
```suggestion
      "I need to get by.",
```

File: data/json/npcs/talk_tags.json [hunk 4]
```diff
@@ -636,7 +667,11 @@
...
+      "Pardon me",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:673]:
```suggestion
      "Pardon me.",
```

File: data/json/npcs/talk_tags.json [hunk 5]
```diff
@@ -656,7 +691,11 @@
       "Thanks, <name_g>!",
       "Pleasure doing business with you.",
       "Now beat it, you <name_b>.",
-      "Good haul.  See you around<punc>"
+      "Good haul.  See you around<punc>",
+      "I need this more than you, I think.",
+      "Thanks for your generosity",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:696]:
```suggestion
      "Thanks for your generosity.",
```

File: data/json/npcs/talk_tags.json [hunk 6]
```diff
@@ -656,7 +691,11 @@
...
+      "Your contribution is much appreciated",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:697]:
```suggestion
      "Your contribution is much appreciated.",
```

File: data/json/npcs/talk_tags.json [hunk 7]
```diff
@@ -656,7 +691,11 @@
...
+      "Your helping to feed my kids"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:698]:
```suggestion
      "You're helping to feed my kids."
```

File: data/json/npcs/talk_tags.json [hunk 8]
```diff
@@ -772,7 +815,13 @@
       "Hands in the air, <name_b>!",
       "Hands up, no sudden movements and we're gravy, okay?",
       "Take it easy and put your hands up now, <name_g>.",
-      "Why don't you put your hands up for me <name_g>."
+      "Why don't you put your hands up for me <name_g>.",
+      "There's some bad folks around these parts, now stick'em up!",
+      "You should have been more careful, now give me your stuff!",
+      "This is a robbery!",
+      "Don't try anything funny",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:822]:
```suggestion
      "Don't try anything funny.",
```

File: data/json/npcs/talk_tags.json [hunk 9]
```diff
@@ -772,7 +815,13 @@
...
+      "Your money or your life!",
+      "Didn't anyone ever tell you not to tlak to strangers?  Give me your shit!"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:824]:
```suggestion
      "Didn't anyone ever tell you not to talk to strangers?  Give me your shit!"
```

File: data/json/npcs/talk_tags.json [hunk 10]
```diff
@@ -811,7 +860,13 @@
       "Consider this idea: you give me food and I eat it.",
       "Did you know that lack of food kills faster than chain smoking?",
       "Can't remember the last time I got a proper meal.",
-      "I could eat a horse."
+      "I could eat a horse.",
+      "fuck, i'm starved",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:864]:
```suggestion
      "<swear>, I'm starved.",
```

File: data/json/npcs/talk_tags.json [hunk 11]
```diff
@@ -811,7 +860,13 @@
...
+      "I could do with a bite to eat right now",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:865]:
```suggestion
      "I could do with a bite to eat right now.",
```

File: data/json/npcs/talk_tags.json [hunk 12]
```diff
@@ -811,7 +860,13 @@
...
+      "I'd kill for some chocolate right now",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:866]:
```suggestion
      "I'd kill for some chocolate right now.",
```

File: data/json/npcs/talk_tags.json [hunk 13]
```diff
@@ -811,7 +860,13 @@
...
+      "Man I miss that burger joint I used to go to",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:867]:
```suggestion
      "Man I miss that burger joint I used to go to.",
```

File: data/json/npcs/talk_tags.json [hunk 14]
```diff
@@ -811,7 +860,13 @@
...
+      "I'm so hungry I can taste it",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:868]:
```suggestion
      "I'm so hungry I can taste it.",
```

File: data/json/npcs/talk_tags.json [hunk 15]
```diff
@@ -830,7 +885,9 @@
       "I've had enough of you, begone",
       "you're a poster child for abortions",
       "how the fuck you've survived this far is beyond me, you <swear> <swear> <name_b>",
-      "you're why the gene pool needs a lifeguard"
+      "you're why the gene pool needs a lifeguard",
+      "Your father should have jerked you off onto a rock and let you dry up in the sun",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:889]:
```suggestion
      "your father should have jerked you off onto a rock and let you dry up in the sun",
```

File: data/json/npcs/talk_tags.json [hunk 16]
```diff
@@ -830,7 +885,9 @@
...
+      "Your a worthless cunt!"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:890]:
```suggestion
      "you're a worthless cunt"
```

File: data/json/npcs/talk_tags.json [hunk 17]
```diff
@@ -846,7 +903,8 @@
       "This sucks.  I'm too big.",
       "I don't like being a mutant crammed into your tiny little vehicle.",
       "Getting awfully cramped in here.",
-      "I don't think this vehicle was designed for someone like me."
+      "I don't think this vehicle was designed for someone like me.",
+      "It's tighter than a nuns cunt in here!'"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:907]:
```suggestion
      "It's tighter than a nun's cunt in here!'"
```

File: data/json/npcs/talk_tags.json [hunk 18]
```diff
@@ -919,7 +979,9 @@
       "<swear!>, a",
       "Look out!  A",
       "<swear!>, here comes a <swear>",
-      "Keep an eye on that"
+      "Keep an eye on that",
+      "I suggest a tactical retreat from that",
+      "Get down! theres a"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:984]:
```suggestion
      "Get down! There's a"
```

File: data/json/npcs/talk_tags.json [hunk 19]
```diff
@@ -995,7 +1063,11 @@
       "Well, that got the blood pumping.",
       "We're clear, but stay frosty.",
       "<swear> beautiful work.",
-      "Getting really good at this."
+      "Getting really good at this.",
+      "Look what you made me do.",
+      "It's better you than me",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1068]:
```suggestion
      "It's better you than me.",
```

File: data/json/npcs/talk_tags.json [hunk 20]
```diff
@@ -995,7 +1063,11 @@
...
+      "Phew, that was close",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1069]:
```suggestion
      "Phew, that was close.",
```

File: data/json/npcs/talk_tags.json [hunk 21]
```diff
@@ -1069,7 +1144,10 @@
       "Is this Reno?  Because I need to watch you die!",
       "You're going to pay for that, <name_b>!",
       "You're gonna rot in hell, you pieces of shit!",
-      "You're gonna rot in hell for this!"
+      "You're gonna rot in hell for this!",
+      "You done fucked up!",
+      "I won't even break a sweat",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1149]:
```suggestion
      "I won't even break a sweat!",
```

File: data/json/npcs/talk_tags.json [hunk 22]
```diff
@@ -1069,7 +1144,10 @@
...
+      "You ain't shit"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1150]:
```suggestion
      "You ain't shit!"
```

File: data/json/npcs/talk_tags.json [hunk 23]
```diff
@@ -1109,7 +1188,14 @@
       "Who's there?",
       "Did anyone else hear that?",
       "Sounds like someone said something.",
-      "Who goes there?"
+      "Who goes there?",
+      "I heard something!",
+      "Huh> what's that",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1193]:
```suggestion
      "Huh, what's that?",
```

File: data/json/npcs/talk_tags.json [hunk 24]
```diff
@@ -1125,7 +1211,12 @@
       "Sounds like something bad's going on.",
       "What's going on over there?",
       "Sounds like a fight's going on.",
-      "What was that?"
+      "What was that?",
+      "Whats happening?",
+      "Somethings happening",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1216]:
```suggestion
      "Something's happening.",
```

File: data/json/npcs/talk_tags.json [hunk 25]
```diff
@@ -1125,7 +1211,12 @@
...
+      "Sounds like another riot",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1217]:
```suggestion
      "Sounds like another riot.",
```

File: data/json/npcs/talk_tags.json [hunk 26]
```diff
@@ -1125,7 +1211,12 @@
...
+      "What was that? Oh god, oh god!",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1218]:
```suggestion
      "What was that?  Oh god, oh god!",
```

File: data/json/npcs/talk_tags.json [hunk 27]
```diff
@@ -1125,7 +1211,12 @@
...
+      "Whats that sound?"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1219]:
```suggestion
      "What's that sound?"
```

File: data/json/npcs/talk_tags.json [hunk 28]
```diff
@@ -1166,7 +1257,19 @@
...
       "Aye aye.",
-      "Aye."
+      "Aye.",
+      "Gotcha.",
+      "Okie Dokie.",
+      "On it.",
+      "No problemo.",
+      "Sounds fun!",
+      "I've got nothing better to do.",
+      "No! just kidding.",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1267]:
```suggestion
      "No!  Just kidding.",
```

File: data/json/npcs/talk_tags.json [hunk 29]
```diff
@@ -1166,7 +1257,19 @@
...
+      "Let's do it.",
+      "I guess I can",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1269]:
```suggestion
      "I guess I can.",
```

File: data/json/npcs/talk_tags.json [hunk 30]
```diff
@@ -1166,7 +1257,19 @@
...
+      "I guess so",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1270]:
```suggestion
      "I guess so.",
```

File: data/json/npcs/talk_tags.json [hunk 31]
```diff
@@ -1166,7 +1257,19 @@
...
+      "I should be able to",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1271]:
```suggestion
      "I should be able to.",
```

File: data/json/npcs/talk_tags.json [hunk 32]
```diff
@@ -1166,7 +1257,19 @@
...
+      "I'll try"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1272]:
```suggestion
      "I'll try."
```

File: data/json/npcs/talk_tags.json [hunk 33]
```diff
@@ -1186,7 +1289,14 @@
       "Man, I can smell the weed, can I have some?",
       "Are you sure it's a good idea to smoke that now?",
       "Are you getting stoned right now?",
-      "Is that the devil's lettuce I smell?"
+      "Is that the devil's lettuce I smell?",
+      "Puff,puff, pass.",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1293]:
```suggestion
      "Puff, puff, pass.",
```

File: data/json/npcs/talk_tags.json [hunk 34]
```diff
@@ -1186,7 +1289,14 @@
...
+      "You got that sticky icky",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1294]:
```suggestion
      "You got that sticky icky.",
```

File: data/json/npcs/talk_tags.json [hunk 35]
```diff
@@ -1186,7 +1289,14 @@
...
+      "Eh yo! lemme hit that!",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1295]:
```suggestion
      "Eh yo!  Lemme hit that!",
```

File: data/json/npcs/talk_tags.json [hunk 36]
```diff
@@ -1186,7 +1289,14 @@
...
+      "roll, roll, roll your joint...",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1296]:
```suggestion
      "Roll, roll, roll your joint…",
```

File: data/json/npcs/talk_tags.json [hunk 37]
```diff
@@ -1216,7 +1326,11 @@
       "Why are you smoking crack cocaine?",
       "Crack cocaine smells awful!",
       "Are you seriously going to be a crackhead around me?",
-      "<meth_smoke>"
+      "<meth_smoke>",
+      "Crack is a hell of a drug",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1330]:
```suggestion
      "Crack is a hell of a drug.",
```

File: data/json/npcs/talk_tags.json [hunk 38]
```diff
@@ -1216,7 +1326,11 @@
...
+      "Can I get a hit?",
+      "Fuckin' Crackhead!",
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1332]:
```suggestion
      "Fuckin' crackhead!",
```

File: data/json/npcs/talk_tags.json [hunk 39]
```diff
@@ -1229,7 +1343,8 @@
       "Hey, <name_g>, can I get some batteries here?  I need to recharge.",
       "Internal batteries running low.  How many batteries can you spare right now?",
       "I can do a lot more for you if you pass me a couple batteries to recharge.",
-      "All these implants I got are going to turn to useless scrap real soon if I don't get some batteries to recharge."
+      "All these implants I got are going to turn to useless scrap real soon if I don't get some batteries to recharge.",
+      "My batteries are all out of juice, need to recharge.'"
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1347]:
```suggestion
      "My batteries are all out of juice, need to recharge."
```

File: data/json/npcs/talk_tags.json [hunk 40]
```diff
@@ -1399,13 +1515,15 @@
...
+      "eldritch horror",
+      "beast",
+      "monstrosity"
     ]
   },
   {
     "type": "snippet",
     "category": "<the_cataclysm>",
-    "text": [ "the Cataclysm", "the apocalypse", "the end of the world", "Armageddon", "the end of days" ]
+    "text": [ "the Cataclysm", "the apocalypse", "the end of the world", "Armageddon", "the end of days", "The end", "the reckoning" ]
```
Comment (Night-Pryanik) [inline data/json/npcs/talk_tags.json:1526]:
```suggestion
    "text": [ "the Cataclysm", "the apocalypse", "the end of the world", "Armageddon", "the end of days", "the end", "the reckoning" ]
```

**Comment by Maleclypse** [comment]
To pass the JSON lint test you need to copy the entire file, paste it into https://dev.narc.ro/cataclysm/format.html then click the lint button. Once it is done in a couple seconds if there were no error messages you copy the entire file from the text box and replace all the text in the json file.  Always do this once you've completed your changes to any json file.

File: data/json/npcs/talk_tags.json (hunk 40 above)
Comment (Stone94) [reply data/json/npcs/talk_tags.json:1526]: lol note to self, proof read.

**Comment by Stone94** [comment]
just figured out how to bulk commit the suggestions, yay!

File: data/json/npcs/talk_tags.json [hunk 41]
```diff
@@ -919,7 +979,9 @@
       "<swear!>, a",
       "Look out!  A",
       "<swear!>, here comes a <swear>",
-      "Keep an eye on that"
+      "Keep an eye on that",
+      "I suggest a tactical retreat from that",
+      "Get down! There's a"
```
Comment (BrettDong) [inline data/json/npcs/talk_tags.json:984]:
```suggestion
      "Get down!  There's a"
```

File: data/json/npcs/talk_tags.json [hunk 42]
```diff
@@ -446,7 +459,9 @@
       "Hey, I'm over here!",
       "Hold up a second, will ya?",
       "What's the rush?",
-      "Wait for me <name_g>, I can't keep up with you like this!"
+      "Wait for me <name_g>, I can't keep up with you like this!",
+      "Where's the rush?",
+      "Hey! Slow down!"
```
Comment (BrettDong) [inline data/json/npcs/talk_tags.json:464]:
```suggestion
      "Hey!  Slow down!"
```

File: data/json/npcs/talk_tags.json [hunk 43]
```diff
@@ -1091,7 +1169,8 @@
       "Watch my back while I stitch my arm back on <name_g>.",
       "Gotta bandage this or I'll bleed out.  Give me a sec.",
       "Let me fix myself up here for a second.",
-      "I ain't got time to ble-wait, that's a lot of blood.  Give me a second while I patch this."
+      "I ain't got time to ble-wait, that's a lot of blood.  Give me a second while I patch this.",
+      "I've been through too much to bleed out now! Bandaging this up!"
```
Comment (BrettDong) [inline data/json/npcs/talk_tags.json:1173]:
```suggestion
      "I've been through too much to bleed out now!  Bandaging this up!"
```

**Pull request merged**